packages/spruce-skill-server/tests/v1APIMocks.js

```diff
@@ -22,6 +22,20 @@ module.exports = ctx => ({
 			})
 			return userLocation
 		}
+
```

Comment:
Is this over-eager? Wouldn't it match deeper paths like `/locations/ID/something/ID`?

Though I suppose v1 is probably fairly locked down to new endpoints now, and this might not be concerning.

Reply:
Oops, off by a line. I'm talking about `path.match(/\/locations\/([^/]+)/)` on line 26.

Reply:
Yeah...good point. I'll update the regex to be more specific